ScoutHelper/Managers/BearManager.cs

```diff
@@ -49,7 +51,7 @@ public class BearManager : IDisposable {
 			throw new Exception("Failed to read in Bear data ;-;");
 		}
 
-		return data
```

Comment:
Sus

Reply:
oh? i'm still returning it, i just wanted to add that log between generating the data and returning it. or did you mean something else? :thinky:

Reply:
Nah my comment was innocuous. I assumed you had made it a variable so you could log the output /capture it in the debugger in the method. Either way, doesn't need any change